pandas/core/tools/datetimes.py

```diff
@@ -400,6 +403,12 @@ def _convert_listlike_datetimes(
             arg = arg.tz_convert(None).tz_localize("utc")
         return arg
 
+    elif isinstance(arg_dtype, ArrowDtype) and arg_dtype.kind == "M":
+        # TODO: Combine with above if DTI/DTA supports Arrow timestamps
+        if utc:
+            arg = arg.astype("timestamp[ns, UTC][pyarrow]")
```

Comment:
I'd rather use ``pd.ArrowDtype(pa...)`` here

Reply:
.astype shouldn't allow converting a tz-naive to tzaware (or vice-versa).  i think this should use tz_convert/tz_localize instead

Reply:
(btw if .astype works on ArrowEA to convert tznaive to tzaware we probably want to change that to match the non-arrow raising behavior)

Reply:
> (btw if .astype works on ArrowEA to convert tznaive to tzaware we probably want to change that to match the non-arrow raising behavior)

Sure, I'll raise a PR after this one gets in.